src/bin/pg_autoctl/cli_enable_disable.c

```diff
@@ -437,6 +437,13 @@ cli_enable_maintenance(int argc, char **argv)
 
 	char *channels[] = { "state", NULL };
 
+	ConnectionRetryPolicy retryPolicy = {
+		POSTGRES_PING_RETRY_TIMEOUT,
+		-1, /* unbounded number of attempts */
+		5 * 1000,               /* sleep up to 5s between attempts */
+		1 * 1000                /* first retry happens after 1 second */
+	};
```

Comment:
We use this same retry policy in 3 places. Can we give it a name that makes sense and make it a constant/function?

Reply:
Good point. The abstraction to set retry policies was a level too high to easily do that, so I changed them to take a ConnectionRetryPolicy instead of a PGSQL argument, and added a new one.